File: 4153/4153.cpp
/* ************************************************************************** */
/*                                                                            */
/*                                                        :::      ::::::::   */
/*   4153.cpp                                           :+:      :+:    :+:   */
/*                                                    +:+ +:+         +:+     */
/*   By: jihoolee <jihoolee@student.42SEOUL.kr>     +#+  +:+       +#+        */
/*                                                +#+#+#+#+#+   +#+           */
/*   Created: 2021/09/30 01:38:49 by jihoolee          #+#    #+#             */
/*   Updated: 2021/09/30 01:49:55 by jihoolee         ###   ########.fr       */
/*                                                                            */
/* ************************************************************************** */

#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

int	main(void)
{
	vector<int>	sides(3);

	while (1)
	{
		cin >> sides[0] >> sides[1] >> sides[2];
		if (sides[0] == 0 && sides[1] == 0 && sides[2] == 0)
			break ;
		sort(sides.begin(), sides.end());
		if (sides[2] * sides[2] == sides[1] * sides[1] + sides[0] * sides[0])
			cout << "right" << endl;
		else
			cout << "wrong" << endl;
	}
	return (0);
}

File: 4153/4153.py
# **************************************************************************** #
#                                                                              #
#                                                         :::      ::::::::    #
#    4153.py                                            :+:      :+:    :+:    #
#                                                     +:+ +:+         +:+      #
#    By: jihoolee <jihoolee@student.42SEOUL.kr>     +#+  +:+       +#+         #
#                                                 +#+#+#+#+#+   +#+            #
#    Created: 2021/09/30 01:50:40 by jihoolee          #+#    #+#              #
#    Updated: 2021/09/30 01:52:50 by jihoolee         ###   ########.fr        #
#                                                                              #
# **************************************************************************** #

while (1):
	sides_lst = list(map(int, input().split()))
	if (sides_lst == [0, 0, 0]):
		break
	sides_lst.sort()
	if (sides_lst[2] ** 2 == sides_lst[1] ** 2 + sides_lst[0] ** 2):
		print('right')
	else:
		print('wrong')
